src/index.html

```diff
@@ -1,13 +1,631 @@
+                intervention in the natural environment.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-brand about-brand--margin-bottom-100">
+        <div class="container container--desktop-padding about-brand__container">
+          <span class="side-text">
+            witch of mountain winds
+          </span>
+          <div class="about-brand__row">
+            <div class="about-brand__item about-brand__info-block">
+              <h2 class="title about-brand__title">
+                about our brand
+              </h2>
+              <p class="about-brand__text">
+                Our brand cosmetics are made from natural ingredients and we are promoting them to the masses. Shampoo,
+                hair
+                conditioner, soap - in recycled paper. Creams, oils, washing products in glass containers with recycled
+                paper labels. No synthetics and microplastics in cosmetics. And the packaging from it can be returned
+                for
+                recycling or reused.
+              </p>
+              <p class="about-brand__text about-brand__more-text-on-desktop">
+                We are all neighbors of one big House. And we must protect what we have. Destroying everything around
+                for the sake of consumerism and imaginary convenience, we destroy what does not belong to us. Be aware
+                of your choice!
+              </p>
+              <a
+                href="#"
+                class="button button--light about-brand__button"
+              >
+                LEARN MORE
+              </a>
+            </div>
+            <div class="about-brand__item about-brand__photo-block">
+              <picture>
+                <source
+                  media="(min-width: 1280px)"
+                  srcset="images/about-img-desktop.png"
+                >
+                <img
+                  src="images/about-img-tablet.png"
+                  alt="about-brand-image"
+                  class="about-brand__big-photo"
+                >
+              </picture>
+              <picture>
+                <img
+                  src="images/about-small-img-desktop.png"
+                  alt="about-brand-image"
+                  class="about-brand__small-photo"
+                >
+              </picture>
+            </div>
+          </div>
+
+        </div>
+      </section>
+      <section
+        class="shop shop--margin-bottom-100"
+        id="shop"
+      >
+        <div class="container container--desktop-padding">
+          <span class="side-text">shop</span>
+          <h2 class="title shop__title">
+            Shop
+          </h2>
+          <div class="shop__category-row">
+            <a
+              href="#"
+              class="shop__category-item _curent"
+            >face</a>
```

Comment:
Need to fix it everywhere
```suggestion
            <a
              href="#"
              class="shop__category-item _curent"
            >
              face
            </a>
```